core/lexicon/en/lexicon.inc.php

```diff
@@ -25,7 +25,8 @@
 $_lang['lexicon_import_err_upload'] = 'An error occurred while trying to upload the Lexicon Topic file. Check your web server permissions in trying to upload to the tmp directory, and validate that the file you are trying to upload is a valid file.';
 $_lang['lexicon_management'] = 'Lexicon Management';
 $_lang['lexicon_management_desc'] = 'Here you can override any Lexicon Entries across Namespaces and their respective Topics. Just double-click on any Entry\'s value to override it. To add new Lexicon Entries or Topics, just create their Topic files in their Namespace\'s respective directory.<br /><b>Each Entry will be available via the [[%name]] placeholder.</b>';
-$_lang['lexicon_rlfb_msg'] = 'Successfully regenerated [[+num]] strings.';
-$_lang['reload_from_base'] = 'Revert All Core Entries';
-$_lang['reload_success'] = 'Successfully reloaded [[+total]] strings.';
+$_lang['lexicon_revert'] = 'Revert Entries';
+$_lang['lexicon_revert_confirm'] = 'Are you sure you want to revert the "[[+namespace]]" namespace > "[[+topic]]" topic > "[[+language]]" lexicon to its default strings?';
+$_lang['lexicon_revert_success'] = 'Reverted [[+total]] total string(s): [[+names]]';
+$_lang['lexicon_revert_error'] = 'No lexicon entries found to revert.';
```

Comment:
1) I'm still in favor of displaying both "[[+namespace]]", and "[[+topic]]", and "[[+language]]" in the message, because pop-up window may cover the selection in the grid and it will not be clear what is selected.
2) @rthrash @smg6511  Can you decide on the final spelling? So that the final version is as clear **or we can leave the current one**?